addons/vehicleCoasting/XEH_PREP.hpp

```diff
@@ -0,0 +1,10 @@
+PREP(addCoastingVehicle);
+PREP(addVehicleHandle);
```

Comment:
Script \z\potato\addons\vehiclecoasting\functions\fnc_addVehicleHandle.sqf not found
Script \z\potato\addons\vehiclecoasting\functions\fnc_dev_trackVehicle.sqf not found
Script \z\potato\addons\vehiclecoasting\functions\fnc_driverDeathPFEH.sqf not found